osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -174,12 +175,15 @@ public MetadataLine(string left, string right)
                         }
                     };
                 }
-
             }
 
+            private readonly SpriteText title;
+            private readonly SpriteText artist;
+            private readonly BeatmapMetadata metadata;
+
             public BeatmapMetadataDisplay(WorkingBeatmap beatmap)
             {
-                var metadata = beatmap?.BeatmapInfo?.Metadata ?? new BeatmapMetadata();
+                metadata = beatmap?.BeatmapInfo?.Metadata ?? new BeatmapMetadata();
 
                 AutoSizeAxes = Axes.Both;
                 Children = new Drawable[]

```

Comment:
you could just move this into the `load()` method to avoid storing references, too.